std/array.d

```diff
@@ -708,8 +708,8 @@ if (isDynamicArray!T && allSatisfy!(isIntegral, I))
 
     auto arr = minimallyInitializedArray!(int[])(42);
     assert(arr.length == 42);
-    // Elements aren't necessarily initialized to 0
-    assert(!arr.equal(0.repeat(42)));
+    // Elements aren't necessarily initialized to 0, so don't do this:
+    // assert(arr.equal(0.repeat(42)));
```

Comment:
Having leftover comment feels a bit sloppy. Could this be turned into a snippet that does the right thing (e.g. initialize the array) ?

Reply:
Good idea